### ISSUE TITLE
tls-wrong-permittedPeer.sh test failure: gnutls error -43: Error in the certificate.

### ISSUE THREAD

**Issue by Whissi** [rsyslog/librelp#113]
```
============================================================
Test: ./tls-wrong-permittedPeer.sh
============================================================
Starting Receiver...
got receive pid 128697
startup wait 0
program started up, pidfile receive.pid contains 128697
Receiver running
Send Message...
send.c: relp engine created new client 0x55d0ba24c3f0
send.c: librelp: CLT permitted peer added: 'wrong name'
send.c: DDDD: gnutls_global_init() called
send.c: librelp: obtained 1 certificates from ./tls-certs/ca.pem
send.c: librelp: auth error: authdata:'', ecode 10036, emsg 'certificate validation failed'
send.c: DDDD: gnutls_handshake: -43: Error in the certificate.
send.c: librelp: generic error: ecode 10031, emsg 'TLS handshake failed [gnutls error -43: Error in the certificate.]'
receive: error 'TLS handshake failed [gnutls error -54: Error in the pull function.]', object 'lstn 30514: conn to clt 127.0.0.1/127.0.0.1'
send.c: end relpSessConnect, iRet 10031
send: authentication error 'certificate validation failed', object ''
send: error 'TLS handshake failed [gnutls error -43: Error in the certificate.]', object 'conn to srvr 127.0.0.1:30514'
failure in: relpCltConnect(pRelpClt, protFamily, port, target)
receive: error 'TLS record write failed [gnutls error -10: The specified session has been invalidated for some reason.]', object 'lstn 30514: conn to clt 127.0.0.1/127.0.0.1'
receiver stopped

FAIL: expected message not found. Expected:
librelp: auth error: authdata:'DNSname: rsyslog; ', ecode 10034, emsg 'no permited name found'

librelp.out.log actually is:
send.c: relp engine created new client 0x55d0ba24c3f0
send.c: librelp: CLT permitted peer added: 'wrong name'
send.c: DDDD: gnutls_global_init() called
send.c: librelp: obtained 1 certificates from ./tls-certs/ca.pem
send.c: librelp: auth error: authdata:'', ecode 10036, emsg 'certificate validation failed'
send.c: DDDD: gnutls_handshake: -43: Error in the certificate.
send.c: librelp: generic error: ecode 10031, emsg 'TLS handshake failed [gnutls error -43: Error in the certificate.]'
send.c: end relpSessConnect, iRet 10031
send: authentication error 'certificate validation failed', object ''
send: error 'TLS handshake failed [gnutls error -43: Error in the certificate.]', object 'conn to srvr 127.0.0.1:30514'
failure in: relpCltConnect(pRelpClt, protFamily, port, target)
FAIL tls-wrong-permittedPeer.sh (exit status: 1)
```

**Comment by Whissi** [comment]
Seems to be fixed in git master...

**Comment by Whissi** [comment]
Ah OK. Tests are failing because certificate used for testing isn't valid anymore. Was fixed via 8491c8a0bcbb89c03875f128ed29baf9c05c4e79.
